### PR TITLE
Fixed typos and improved style of css tutorial

### LINKED ISSUE #1343: Please fix spelling...
Please fix the spelling error in "Add a declaration to an existing rulest in DevTools" - it should be "ruleset", not "rulest".

- It is an excellent tutorial, thank you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0d236a87-dbb2-6330-23a8-c88eb641c12a
* Version Independent ID: 514f72b6-b104-ee3e-a982-98510467422e
* Content: [DevTools for beginners: Get started with CSS - Microsoft Edge Development](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/beginners/css)
* Content Source: [microsoft-edge/devtools-guide-chromium/beginners/css.md](https://github.com/MicrosoftDocs/edge-developer/blob/master/microsoft-edge/devtools-guide-chromium/beginners/css.md)
* Product: **microsoft-edge**
* Technology: **devtools**
* GitHub Login: @MSEdgeTeam
* Microsoft Alias: **msedgedevrel**

### PR DESCRIPTION
**Pull request by captainbrosset**
There were a couple of typos on this page that I fixed.
I used this opportunity to improve the score reported by acrolinx (from 89 to 95), by changing some of the style.

Fixes #1343.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/beginners/css.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ keywords: microsoft edge, web developement, f12 tools, devtools
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/beginners/css.md:26]: In the screenshot that's below "Choose New Style Rule (New Style Rule) which is next to .cls. An empty ruleset appears with a as the selector.", add red box around +.   eg: 
1. in VS Code, with doc tool extension installed, click the image link to display image in VS Code.  
2. shortcut Ctrl+Alt+R opens File Explorer w/ the file selected.
3. Right-click, Edit (in Paint).  
4. Zoom (Ctrl+PgUp).  3px wide red box. 
5. Save.
I jumped into that section w/o context, and it took a long time to figure out what the sentence was talking about in the screenshot.  Good example of why red boxes are required, for dense UI.

File: microsoft-edge/devtools-guide-chromium/beginners/css.md [hunk 2]
```diff
@@ -590,11 +590,11 @@ Complete the following actions to bootstrap and provide classes for aligning ele
     
 ## Next steps  
 
-Congratulations, you are done.  
+Congratulations, you're done.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/beginners/css.md:593]:
```suggestion
Congratulations, you're done!
```
Suggested for natural tone in this context.  https://docs.microsoft.com/en-us/style-guide/punctuation/exclamation-points

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Edit pass.  Suggested changes, largely to make the tone more natural.

**Comment by captainbrosset** [comment]
Thank you @mikehoffms, I believe I addressed all your comments. I agree the tone of this tutorial was very robotic.
I also changed one of the class names used in the code from `custom-background` to `my-class-name` to make it more obvious that this is up to users to choose, and to avoid making the class name too specifically tied to what it actually changes.

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**